cvmfs/swissknife_history.cc

```diff
@@ -40,6 +40,7 @@ void CommandTag::InsertCommonParameters(ParameterList *r) {
   r->push_back(Parameter::Optional(
     'e', "hash algorithm to use (default SHA1)"));
   r->push_back(Parameter::Switch('L', "follow HTTP redirects"));
+  r->push_back(Parameter::Switch('Z', "Compression Algorithm"));
```

Comment:
This is supposed to be an optional parameter, no? `Parameter::Switch()` doesn't assume a string parameter after `-Z`. Probably you want to use `Parameter::Optional()` here.